pages/requester/create-task.tsx

```diff
@@ -212,14 +249,18 @@ export default function Form() {
               </Grid>
             </Grid>
           </FormControl>
-
+          <Typography color="primary" mt={2} mx="auto">
+            Taskers will earn {process.env.NEXT_PUBLIC_UNIT_ETH_REWARD} ETH per
+            completed task. This reward is subject to change.
```

Comment:
We should dynamically update the amount earned per completed task based on maxReward/maxTaskers if we're going with non-fixed rewards

Reply:
done